packages/website/docs/_components_pages/fiori/SideNavigation/SideNavigation.mdx

```diff
@@ -10,6 +11,9 @@ import QuickAction from "../../../_samples/fiori/SideNavigation/QuickAction/Quic
 
 ## More Samples
 
+### Unselectable (One-Click Area) Parent Items
```

Comment:
It looks better to call this just "Unselectable Parent Items"
And add one sentence below the title to explain 
"When selecting the parent item it will expand or collapse the list of child items and will not get selected."

Reply:
Done